src/main.qml

```diff
@@ -894,7 +894,9 @@ ApplicationWindow {
                                 ? qsTr("Cached on your computer")
                                 : url.startsWith("file://")
                                   ? qsTr("Local file")
-                                  : qsTr("Online - %1 GB download").arg((image_download_size/1073741824).toFixed(1))
+                                  : image_download_size >= 1073741824
+                                  ? qsTr("Online - %1 GB download").arg((image_download_size/1073741824).toFixed(1))
```

Comment:
Thinking on this further - given we're reusing GB/MB across languages, perhaps we could make GB/MB and untranslated parameter to this string, and just prune it from the translations?